src/components/Counter/Counter.st.css

```diff
@@ -46,6 +46,11 @@
     border-color: color(value(counterColor));
 }
 
+.root.xsmall {
+    padding: 0 4px;
+    width: 64px;
```

Comment:
I think the `width` should probably also be overridable, as in line 40

Reply:
tried, the following didn't work, wdim?
`:vars {
    /*Defaults*/
    ...
    DefaultWidthXsmall: 64px;
}

:vars {
    ...
    counterWidthXsmall: fallback(value(MainWidth), value(counterWidthXsmall));
}

.xsmall {
    width: value(counterWidthXsmall);
}`